odoo/addons/base/models/res_users.py

```diff
@@ -667,7 +667,7 @@ def context_get(self):
 
     @tools.ormcache('self._uid')
     def _get_company_ids(self):
-        return self.company_ids.ids
+        return frozenset(self.company_ids.ids)
```

Comment:
Not in the right commit. (Should be in https://github.com/odoo/odoo/pull/88276/commits/98d596de7f2ef08e518a1b366a50f1f7a34a2e12)